spec/models/story_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Story, type: :model do
+  describe '#tagged_stories' do
+    it 'should return one story' do
+      FactoryGirl.create(:story, tags: ['NDC'])
+      expect(Story.tagged_stories('NDC, esp, climate watch', 5)).to have(1).items
+    end
+
+    it 'should not return any story' do
+      FactoryGirl.create(:story, tags: ['OP23'])
+      expect(Story.tagged_stories('NDC, esp', 5)).to have(0).items
+    end
+  end
+
+  describe '#untagged_stories' do
+    it 'should return no story' do
+      FactoryGirl.create(:story)
```

Comment:
This test is not explicit enough in my opinion. Because it's trusting the factory to create the story with tags, it's not transparent that that's the case. Given that we are testing if it will return only untagged stories, I would add the tags information to the FactoryGirl create method.

Reply:
And now that I read the test bellow, I'm a bit confused, sorry.

So my real comment is. This test with "0" as an argument to `untagged_stories` is in fact testing the ActiveRecord `limit` method. I'd suggest that the test would create some tagged stories and then try to fetch untagged stories and have none be returned.

Reply:
Maybe the name of the method isn't adequate enough. It should return stories, tagged or not, that not match the tags that have been passed as query params

Reply:
Yes, that is a bit misleading. So maybe it could be called `not_tagged_by(my_tags)`